patroni/postgresql/bootstrap.py

```diff
@@ -116,22 +116,31 @@ def _custom_bootstrap(self, config):
             self._postgresql.config.remove_recovery_conf()
         return True
 
+    def get_local_connect_kwargs(self):
+        """
+        get local connect kwargs and pgpass
+        returns tuple (connectString, env)
+        """
+        r = self._postgresql.config.local_connect_kwargs
+        connstring = self._postgresql.config.format_dsn(r, True)
+        if 'host' not in r:
+            # https://www.postgresql.org/docs/current/static/libpq-pgpass.html
+            # A host name of localhost matches both TCP (host name localhost) and Unix domain socket
+            # (pghost empty or the default socket directory) connections coming from the local machine.
+            r['host'] = 'localhost'  # set it to localhost to write into pgpass
+
+        env = self._postgresql.config.write_pgpass(r)
+        env['PGOPTIONS'] = '-c synchronous_commit=local'
+
+        return connstring, env
+
```

Comment:
This method is called from HA in order to activate liveness check, which has nothing to do with a bootstrap, hence I think this method should be created in the Postgresql() class.

Reply:
Yet another concern is that the health check will break if someone removes pgpass file.